scripts/hurricane/stats/exevs_hurricane_regional_tropcyc_stats.sh

```diff
@@ -130,9 +130,30 @@ sed -i "s|$SEARCHx|$Model_List|g" TCPairs_template.conf
 
 run_metplus.py -c $STORMdata/TCPairs_template.conf
 
+#---run for TC_RMW      
+cp ${PARMevs}/TCRMW_template.conf .
+export SEARCH0="METBASE_template"
+export SEARCH1="INPUT_BASE_template"
+export SEARCH2="OUTPUT_BASE_template"
+export SEARCH3="INIT_BEG_template"
+export SEARCH4="INIT_END_template"
+export SEARCH5="TC_RMW_CYCLONE_template"
+export SEARCH6="TC_PAIRS_BASIN_template"
+export SEARCHx="MODELLIST_template"
+sed -i "s|$SEARCH0|$MetOnMachine|g" TCRMW_template.conf
+sed -i "s|$SEARCH1|$STORMdata|g" TCRMW_template.conf
+sed -i "s|$SEARCH2|$STORMroot|g" TCRMW_template.conf
+sed -i "s|$SEARCH3|$startdate|g" TCRMW_template.conf
+sed -i "s|$SEARCH4|$enddate|g" TCRMW_template.conf
+sed -i "s|$SEARCH5|$stormNumber|g" TCRMW_template.conf
+sed -i "s|$SEARCH6|$stbasin|g" TCRMW_template.conf
+sed -i "s|$SEARCHx|$Model_List|g" TCRMW_template.conf
+
+run_metplus.py -c $STORMdata/TCRMW_template.conf
+
 #--- run for TC_stat 
 cd $STORMdata
-cp ${PARMevs}/metplus_config/hurricane/stats/TCStat_template.conf .
+cp ${PARMevs}/TCStat_template.conf .
 
```

Comment:
You will want to get this change back into your PR. You accidentally introduced older code for line 135.